src/main/java/uk/ac/cam/cl/dtg/segue/api/AuthorisationFacade.java

```diff
@@ -289,20 +323,54 @@ public Response releaseAllAssociations(@Context final HttpServletRequest request
     }
 
     /**
-     * Get all users whose data I can see.
-     * 
+     * Get all users whose data the current user can see.
+     *
      * @param request
      *            - so we can identify the current user.
+     * @throws NoUserException
+     *            - when the target user cannot be found.
      * @return List of user associations.
      */
     @GET
     @Path("/other_users")
     @Produces(MediaType.APPLICATION_JSON)
     @GZIP
     @ApiOperation(value = "List all users the current user has been granted access by.")
-    public Response getCurrentAccessRights(@Context final HttpServletRequest request) {
+    public Response getCurrentAccessRights(@Context final HttpServletRequest request) throws NoUserException {
         try {
-            RegisteredUserDTO user = userManager.getCurrentRegisteredUser(request);
+            RegisteredUserDTO requestingUser = userManager.getCurrentRegisteredUser(request);
+            return getCurrentAccessRightsSpecificUser(request, requestingUser.getId());
+        } catch (NoUserLoggedInException e) {
+            return SegueErrorResponse.getNotLoggedInResponse();
+        }
+    }
+
+    /**
+     * Get all users whose data the specified user can see.
+     * 
+     * @param request
+     *            - so we can identify the current user.
+     * @param userId
+     *            - the id of the user we want the access rights of.
+     * @throws NoUserException
+     *            - when the target user cannot be found.
+     * @return List of user associations.
+     */
+    @GET
+    @Path("/other_users/{userId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @GZIP
+    @ApiOperation(value = "List all users the specified user has been granted access by.")
+    public Response getCurrentAccessRightsSpecificUser(@Context final HttpServletRequest request, @PathParam("userId") Long userId) throws NoUserException{
```

Comment:
Stylistic pet-peeve here, feel free to ignore me, I'm just being a grumpy old man, but I'd rather rename this to something like `getCurrentAccessRightsForSpecificUser` or even `getCurrentAccessRightsForUser`, but maybe Apple led me astray with their Smalltalk-inspired messages they used in their Objective-C Cocoa API.